common/app/common/NewNavigation.scala

```diff
@@ -0,0 +1,83 @@
+package common
+
+sealed class TopLevelSection(val name: String)
+
+case object News extends TopLevelSection("news")
+case object Opinion extends TopLevelSection("opinion")
+case object Sports extends TopLevelSection("sports")
+case object Arts extends TopLevelSection("arts")
+case object Life extends TopLevelSection("life")
```

Comment:
@TBonnin yes that makes sense. So in the end it would look like this: 

```

sealed class TopLevelSection(val name: String)

object TopLevelSection {
  case object News extends TopLevelSection("news")
  case object Opinion extends TopLevelSection("opinion")
  case object Sports extends TopLevelSection("sports")
  case object Arts extends TopLevelSection("arts")
  case object Life extends TopLevelSection("life")
}
```